### ISSUE TITLE
Script fails with spaces in sql password

### ISSUE THREAD

**Issue by meichthys** [flexcomng/erpnext_quick_install#44]
```
Error: Got unexpected extra arguments (my password here)
An error occurred on line 369 with exit status 0
```

**Comment by RaySolo** [comment]
just change the password parameter into "${password}". that should help.

**Comment by flexcomng** [comment]
> just change the password parameter into "${password}". that should help.

@meichthys did the suggestion by @RaySolo help solve your error?

**Comment by fthobe** [comment]
Yes, it does work, but passwords shouldn't contain spaces as many systems struggle with them.

**Comment by flexcomng** [comment]
I don't encourage using spaces in passwords as it throws up all kinds of unique problems so i will not be implementing this.
